frontend/javascripts/oxalis/model/sagas/mesh_saga.ts

```diff
@@ -847,19 +847,21 @@ function* loadPrecomputedMeshForSegmentId(
     return;
   }
 
-  yield* call(
-    _getLoadChunksTasks,
-    dataset,
-    layerName,
-    meshFile,
-    segmentationLayer,
-    id,
-    seedPosition,
-    availableChunksMap,
-    loadingOrder,
-    scale,
-    additionalCoordinates,
-  );
+  for (const lod of loadingOrder) {
```

Comment:
Why did you move up the for loop? Previously this was done in in loadPrecomputedMeshesInChunksForLod / _getLoadChunksTasks

Reply:
the `lod` loop variable was used within the for-loop-body inside an async task (closure). it wasn't clear to me whether the closure would always access the correct value (whether it would depend on the loop execution). to exclude this error class, I refactored it.